internal/indexer/indexer_utils.go

```diff
@@ -80,44 +76,44 @@ func CheckBounds(ranges indexer.QueryRange, v interface{}) bool {
 		if lowerBound != nil {
 			cmp, isFloat, err := compareInt(vVal, lowerBound)
 			if err != nil {
-				return false
+				return false, err
 			}
 			if cmp == -1 || (isFloat && cmp == 0 && !ranges.IncludeLowerBound) {
-				return false
+				return false, err
 			}
 		}
 		if upperBound != nil {
 			cmp, isFloat, err := compareInt(vVal, upperBound)
 			if err != nil {
-				return false
+				return false, err
 			}
 			if cmp == 1 || (isFloat && cmp == 0 && !ranges.IncludeUpperBound) {
-				return false
+				return false, err
 			}
 		}
 
 	case *big.Float:
 		if lowerBound != nil {
 			cmp, isFloat, err := compareFloat(vVal, lowerBound)
 			if err != nil {
-				return false
+				return false, err
 			}
 			if cmp == -1 || (cmp == 0 && isFloat && !ranges.IncludeLowerBound) {
-				return false
+				return false, err
 			}
 		}
 		if upperBound != nil {
 			cmp, isFloat, err := compareFloat(vVal, upperBound)
 			if err != nil {
-				return false
+				return false, err
 			}
 			if cmp == 1 || (cmp == 0 && isFloat && !ranges.IncludeUpperBound) {
-				return false
+				return false, err
 			}
 		}
 
 	default:
-		return false
+		return false, fmt.Errorf("invalid argument type in query")
```

Comment:
Please add the arg's contents in the error message to improve experience in potential troubleshooting of this code